tests/start_network.py

```diff
@@ -159,24 +156,14 @@ def add(parser):
             "--snapshot-dir",
             help="Snapshot directory to recover from (optional)",
         )
-        parser.add_argument(
-            "--network-enc-pubk",
-            help="Defunct network public encryption key (used by members to decrypt recovery shares)",
-        )
         parser.add_argument(
             "--common-dir",
-            help="Directory containing previous network member identities and network encryption key",
+            help="Directory containing previous network member identities",
         )
 
     args = infra.e2e_args.cli_args(add)
-    if args.recover and (
-        args.ledger_dir is None
-        or args.common_dir is None
-        or args.network_enc_pubk is None
-    ):
-        print(
-            "Error: --recover requires --ledger-dir, --network-enc-pubk and --common-dir arguments."
-        )
+    if args.recover and (args.ledger_dir is None or args.common_dir is None):
```

Comment:
```suggestion
    if args.recover and not all(args.ledger_dir, args.common_dir):
```

Not strictly equivalent, but empty paths aren't valid, and arguably more readable.